setup.py

```diff
@@ -6,7 +6,7 @@
     long_description = fh.read()
 
 # Parse requirements.txt
-with open("requirements.txt") as f:
+with open("requirements-base.txt") as f:
```

Comment:
all in all, i like packaging it up, but we are not really publishing outside of the docker container. maybe it makes more sense to  create a wheel of the requirements, install the wheel, and then just copy the scripts?

Reply:
I don't disagree, but keeping it an installable makes for faster dev loops IMO, and it doesn't add much complexity at all. Furthermore it is easier to use outside docker if needed.